.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,25 @@
+name: Release Workflow
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up JDK 11
+        uses: actions/setup-java@v4
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          cache: 'sbt'
+      - name: Build tarball
```

Comment:
since this is release workflow testing.. lets also test with `build/sbt publishLocal` works. That will test out the whole publishing workflow as well.